src/targeting.js

```diff
@@ -240,13 +240,13 @@ export function newTargeting(auctionManager) {
     let targetingMap = Object.keys(targetingCopy).map(adUnitCode => {
       return {
         adUnitCode,
-        adUnitTargeting: targetingCopy[adUnitCode]
+        adserverTargeting: targetingCopy[adUnitCode]
```

Comment:
curious why this name was changed?

Reply:
The sortByDealAndPriceBucketOrCpm is now also used in getHighestCpmBidsFromBidPool and the bids received object has the original name of adserverTargeting. If we want to leave the adUnitTargeting name for that cloned targeting map I could change it to work a different way. Just felt it made sense to go with the same name and went with the original adserverTargeting name.